jerry-core/ecma/operations/ecma-objects-arguments.cpp

```diff
@@ -186,14 +167,11 @@ ecma_op_create_arguments_object (ecma_object_t *func_obj_p, /**< callee function
         {
           ecma_string_t *indx_string_p = ecma_new_ecma_string_from_uint32 ((uint32_t) indx);
 
-          prop_desc = ecma_make_empty_property_descriptor ();
```

Comment:
Because we should use a ```property_descriptor``` out of ```if (formal_params_p != NULL)``` block, in [L232](https://github.com/Samsung/jerryscript/pull/684/files?diff=unified#diff-c21c3dee8026ba5fc844a73046372a83R232).